odl/discr/discretization.py

```diff
@@ -391,12 +391,15 @@ def interpolation(self):
 
     def __str__(self):
         """Return ``str(self)``."""
-        return arraynd_str(self.asarray())
+        return array_str(self)
 
     def __repr__(self):
         """Return ``repr(self)``."""
-        return '{!r}.element({})'.format(self.space,
-                                         arraynd_repr(self.asarray()))
+        if self.ndim == 1 and self.size <= 6:
```

Comment:
There are numpy variables for this, `numpy.get_printoptions()['threshold']`, not at all sure if we want to use it.

Reply:
Dunno either, I added a tool to temporarily set them, so it may be a good way to have the caller set the option and let it trickle through like that.